### PR TITLE
py-intensity-normalization: add package (at version 2.1.1)

### PR DESCRIPTION
**Pull request by bcdarwin**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-intensity-normalization/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class PyIntensityNormalization(PythonPackage):
+    """Normalize intensities of images from various MRI modalities"""
+
+    homepage = "https://github.com/jcreinhold/intensity-normalization"
+    pypi     = "intensity-normalization/intensity-normalization-2.1.1.tar.gz"
+
+    version('2.1.1', sha256='686b86754a9a520a03f793cb15c87e945f68ede78ac0ad1b3564c5d5b7ac9486')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intensity-normalization/package.py:17]: Missing build dep on setuptools and pytest-runner

File: var/spack/repos/builtin/packages/py-intensity-normalization/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on('py-matplotlib')
+    depends_on('py-nibabel')
+    depends_on('py-numpy')
+    depends_on('py-scikit-fuzzy')
+    depends_on('py-scikit-image')
+    depends_on('py-scikit-learn')
+    depends_on('py-scipy')
+    depends_on('py-statsmodels')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intensity-normalization/package.py]: These deps should be `type=('build', 'run')` too

File: var/spack/repos/builtin/packages/py-scikit-fuzzy/package.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+class PyScikitFuzzy(PythonPackage):
+    """Fuzzy logic toolkit for SciPy"""
+
+    homepage = "https://github.com/scikit-fuzzy/scikit-fuzzy"
+    pypi     = "scikit-fuzzy/scikit-fuzzy-0.4.2.tar.gz"
+
+    version('0.4.2', sha256='1ab12424d847ede1bc79670d8058167be7c8dd660b00756e9b844817ceb1e12e')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-scikit-fuzzy/package.py:17]: Missing a build dep on setuptools

File: var/spack/repos/builtin/packages/py-scikit-fuzzy/package.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    depends_on('py-networkx')
+    depends_on('py-numpy')
+    depends_on('py-scipy')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-scikit-fuzzy/package.py]:
```suggestion
    depends_on('py-networkx@1.9:', type=('build', 'run'))
    depends_on('py-numpy@1.6:', type=('build', 'run'))
    depends_on('py-scipy@0.9:', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by bcdarwin** [comment]
@adamjstewart - all suggestions applied, thanks.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**